test/example_test.go

```diff
@@ -0,0 +1,53 @@
+/*
+Go has a testing package from the stdlib. But here
+we are also using Testify - a third-party package to
+provide more options. This may change in the future.
+*/
+package example_test
+
+import (
+	 example "github.com/cisagov/skeleton-golang-package/src"
+	"github.com/stretchr/testify/assert"
+	"testing"
```

Comment:
Group imports by type (stdlib, external, internal). This aligns with how [goimports](https://pkg.go.dev/golang.org/x/tools/cmd/goimports) styles imports.

```suggestion
	"testing"

	"github.com/stretchr/testify/assert"

	example "github.com/cisagov/skeleton-golang-package/src"
```